frontend/views/components/Modal/Modal.vue

```diff
@@ -48,7 +48,7 @@ export default {
       } else {
         // When the route change we compare to see if the modal changed
         // If so, we send the event to close the modal
-        if (from.query.modal) {
+        if (this.content && this.content.length) {
```

Comment:
I'm not sure I understand this line, can you explain? What was the reason for this change? I remember adding the check `from.query.modal` to prevent a bug before.

Unrelated note: checking `this.content.length` is unnecessary since `this.content` is either a string or null, and both `''` and `null` are falsy

Reply:
What bug did you prevent to happen?
Here, we are forcing the modal to close if there isn't a modal content to show to prevent this bug where you click to close a modal at the same time you are asking to redirect a page (logout): So, sometimes the logout and redirect would happen before the modal closes, and in that moment `from.query.modal` doesn't exist anymore. That's why the modal didn't close.

![image](https://user-images.githubusercontent.com/14869087/67272932-2bf9bd80-f4c6-11e9-8a11-5bb4563d9609.png)